site/content/posts/2021-04-13-Velero-1.6-Bring-All-Your-Credentials.md

```diff
@@ -0,0 +1,109 @@
+
+### Increased Stability with End-to-end Testing
+
+As the Velero project continues to grow, it is important that our community of users can continue to trust and rely on Velero to support them in their disaster recovery and migration needs.
+During this release cycle, we decided to focus on increased stability by adding a new suite of [end-to-end tests](https://github.com/vmware-tanzu/velero/tree/main/test/e2e).
+Although Velero features have always been thoroughly tested, these new tests execute Velero workflows from the perspective of the user interacting with a Velero installation using the CLI.
+This new test framework has enabled us to create tests that can be easily run across multiple Kubernetes providers, ensuring that we prevent regressions quickly and early in the development of new features.
+
+We plan to expand on this work in our upcoming releases, including [adding more test cases](https://github.com/vmware-tanzu/velero/issues/3531) and running these tests as [part of our CI process](https://github.com/vmware-tanzu/velero/issues/3574).
+
+### Growing the Velero Team
+
+We're excited to announce that [JenTing Hsiao](https://github.com/jenting) from SUSE has joined the Velero team as a maintainer! Already a maintainer of the Velero Helm chart, JenTing has consistently contributed to the Velero community and we’d like to thank him for his continued support to the project and community.
+Welcome, JenTing!
+
+We are also glad to announce that [Eleanor Millman](https://github.com/eleanor-millman) from VMware has joined the project as our product manager.
+She is working with the team and community to develop the Velero vision and roadmap.
+Feel free to reach out to her with any product thoughts or feedback.
+Welcome, Eleanor!
+
+The Velero project welcomes contributors of all kinds.
+If you are interested in contributing more, or finding out what is involved in becoming a maintainer, please see our documentation on [project governance](https://github.com/vmware-tanzu/velero/blob/main/GOVERNANCE.md).
+
+### Improving Developer Experience
+
+As we continue to grow our community of contributors, we want to lower the barrier to entry for making contributions to the Velero project.
+We’ve made huge improvements to the developer experience during this release cycle by introducing Tilt to the developer workflow.
+Using Tilt enables developers to make changes to Velero and its plugins, and have those changes automatically built and deployed to your cluster.
+This removes the need for any manual building or pushing of images, and provides a faster and much simpler workflow.
+Our Tilt configuration also enables contributors to more easily debug the Velero process using Delve, which has integrations with many editors and IDEs.
+If you would like to try it out, please see our [documentation](https://velero.io/docs/v1.6/tilt/).o
```

Comment:
This `.o` at the end of the paragraph.

Reply:
:woman_facepalming: haha, I don't know how I added that! Fixed.